Libplanet.Net.Tests/Consensus/ConsensusReactorTest.cs

```diff
@@ -46,21 +48,21 @@ public void Dispose()
                 Array.Empty<IceServer>(),
                 null);
 
-            var messageTransport = new NetMQTransport(
+            var consensusTransport = new NetMQTransport(
                 key,
                 TestUtils.AppProtocolVersion,
                 null,
                 8,
                 host,
-                port + 1000,
```

Comment:
I guess we need some defensive about out of 65535. Or just use static numeric like 6100 or 7000

Reply:
Amended!